src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -16,6 +16,81 @@ def filter_notification_link(link_text, amount, filter_item)
     end
   end
 
+  def notification_icon(notifiable_type)
+    return image_tag('request_icon', size: 18) if notifiable_type == 'BsRequest'
+
+    capture do
+      tag.i(class: 'fas fa-comments mr-1')
+    end
+  end
+
+  def request_state(notification)
+    state = if notification.notifiable_type == 'BsRequest'
+              notification.notifiable.state
+            elsif notification.notifiable_type == 'Review'
```

Comment:
what will happen if notifiable_type is neither `BsRequuest` or `Review`?

Reply:
We have slightly modified the code, so it does nothing in case it's not `BsRequest`. Review part is dropped since it's going to disappear soon.